openfisca_france/parameters/impot_revenu/abattements_rni/personne_agee_ou_invalide/montant_2.yaml

```diff
@@ -0,0 +1,9 @@
+description: Âgé ou invalide - montant
+reference:
+  - https://www.ipp.eu/baremes-ipp/impot-sur-le-revenu/calcul_revenus_imposables/abat_rni/
+  - http://bofip.impots.gouv.fr/bofip/2036-PGP.html
+  - https://www.legifrance.gouv.fr/affichCodeArticle.do?cidTexte=LEGITEXT000006069577&idArticle=LEGIARTI000006308322
+unit: currency
+values:
+  2018-01-01:
+    value: 1188.0
```

Comment:
Il me semble que 1188 c'est la valeur en 2017 tandis que celle en 2018 est 1208 (cf. [ici](http://bofip.impots.gouv.fr/bofip/2036-PGP) et [ici](https://framagit.org/french-tax-and-benefit-tables/baremes-ipp-yaml/blob/update-bareme-impot-revenu/parameters/impot_revenu/calcul_revenus_imposables/abat_rni.yaml))